Source/JavaScriptCore/b3/B3LowerToAir.cpp

```diff
@@ -3986,9 +4107,22 @@ class LowerToAir {
             return;
         }
         case B3::VectorExtaddPairwise:
+            if (isX86()) {
+                SIMDValue* value = m_value->as<SIMDValue>();
+                if (value->simdLane() == SIMDLane::i16x8 && value->signMode() == SIMDSignMode::Unsigned)
+                    append(VectorExtaddPairwiseUnsignedInt16, tmp(value->child(0)), tmp(value), m_code.newTmp(B3::FP));
```

Comment:
Why can't we just always emit VectorExtaddPairwise, and selectively lower in lowerMacros? We can make tmps there too.

Reply:
I just copied our implementation from `AirIRGenerator`, but good point! We can probably remove this from here and from Air and replace both with a lower macro.

Reply:
Updated.